.github/workflows/lint.yaml

```diff
@@ -67,6 +67,11 @@ jobs:
       - name: Run Helm linter
         run: |
           helm lint ${{ env.HELM_CHARTS_DIR }}
+      - name: Run TypeSpec format check
+        run: |
+          pushd typespec
+          npx tsp format --check "**/*.tsp"
+          popd
```

Comment:
What does the failure message look like when this fails? Is it something a user could figure out?

Reply:
![image](https://github.com/radius-project/radius/assets/5220939/651b4bef-6b54-4a50-80c7-bc0607c99d24)

Reply:
It might be helpful to do something like this. 

```
npx tsp format --check "**/*.tsp" || { echo "TypeSpec format check failed. Run 'npx tsp format **/*.tsp' and try again."; exit 1 }
```